mapie_v1/classification.py

```diff
@@ -52,6 +49,7 @@ def predict(self, X: ArrayLike) -> NDArray:
         scikit-learn classifiers
         Shape (n_samples,)
         """
+        pass
 
     def predict_sets(
```

Comment:
Missing Parameter `X: ArrayLike`

Reply:
Already here (not indented) line 44, isn't it?
`def predict(self, X: ArrayLike)`